example_config.yaml

```diff
@@ -0,0 +1,78 @@
+# Top-level parameters
+
+exp-name: E-MTAB-1234
+droplet: False
```

Comment:
maybe instead of having a droplet true / false we could have a experiment type or technology type? with some predefined acceptable values?

Reply:
Only the droplet status is strictly necessary for this mechanism to work, because it impacts how the proto-MAGETAB files will be generated. But obviously we can record the technology type if it's import to display.